.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
For some reason `next lint` didn't pick up the `.js` configuration, but we don't actually need it anyway.